src/vstest.console/Internal/ConsoleLogger.cs

```diff
@@ -61,7 +61,8 @@ internal enum Verbosity
         {
             Quiet,
             Minimal,
-            Normal
+            Normal,
+            Detailed
```

Comment:
Add detailed to [help](https://github.com/Microsoft/vstest/blob/01ad93c84bd5afcc5a521b80049527d52fa25324/src/vstest.console/Resources/Resources.resx#L241) for customer to aware this option.